go/chat/server.go

```diff
@@ -1675,6 +1651,10 @@ func (h *Server) DownloadAttachmentLocal(ctx context.Context, arg chat1.Download
 	ctx = Context(ctx, h.G(), arg.IdentifyBehavior, &identBreaks, h.identNotifier)
 	defer h.Trace(ctx, func() error { return err }, "DownloadAttachmentLocal")()
 	defer func() { err = h.handleOfflineError(ctx, err, &res) }()
+	if err = h.assertLoggedIn(ctx); err != nil {
```

Comment:
why not `h.assertLoggedInUID`?

Reply:
copy/pasted

Reply:
Actually that code has been there forever, if I changed it wasn't anything significant.

Reply:
oh ok i just noticed we never use `assertLoggedInUID` throughout the file even when we also need a uid and was wondering why